src/skydb.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const FormData = require("form-data");
+const fs = require("fs");
+const p = require("path");
+
+const { Upload } = require("tus-js-client");
+
+const { buildRequestHeaders, SkynetClient } = require("./client");
+const { defaultOptions, getFileMimeType, makeUrl, walkDirectory, uriSkynetPrefix } = require("./utils");
+
+/**
+ * The tus chunk size is (4MiB - encryptionOverhead) * dataPieces, set in skyd.
+ */
+const TUS_CHUNK_SIZE = (1 << 22) * 10;
+
+/**
+ * The retry delays, in ms. Data is stored in skyd for up to 20 minutes, so the
+ * total delays should not exceed that length of time.
+ */
+const DEFAULT_TUS_RETRY_DELAYS = [0, 5000, 15000, 60000, 300000, 600000];
+
+const defaultUploadOptions = {
+  ...defaultOptions("/skynet/skyfile"),
+  endpointLargeUpload: "/skynet/tus",
+
+  portalFileFieldname: "file",
+  portalDirectoryFileFieldname: "files[]",
+  customFilename: "",
+  customDirname: "",
+  dryRun: false,
+  errorPages: undefined,
+  largeFileSize: TUS_CHUNK_SIZE,
+  retryDelays: DEFAULT_TUS_RETRY_DELAYS,
+  tryFiles: undefined,
+};
+
+const nodejs_db_setJSON = async function (privateKey, dataKey, data, customOptions = {}) {
+    const opts = { ...defaultUploadOptions, ...this.customOptions, ...customOptions };
+    const SkydbFormatedData = '{"_data":' + JSON.stringify(data) + ',"_v":2}';
```

Comment:
```suggestion
    const SkydbFormattedData = '{"_data":' + JSON.stringify(data) + ',"_v":2}';
```

Reply:
Can you extract out the `2` into a constant `JSON_RESPONSE_VERSION`?

Reply:
I saw too late, I have already changed it in the new version. Sorry.

Reply:
I simply integrated the original function "buildSkynetJsonObject(data)" from "skynet-js" from "skydbV2.ts", rewritten for nodejs! I use it in my new version of "skydb.js" and "skydbV2.js"!
New version is not on github yet. But she's coming soon. I give notice.